src/H5FDcore.h

```diff
@@ -25,8 +25,70 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-H5_DLL hid_t  H5FD_core_init(void);
+H5_DLL hid_t H5FD_core_init(void);
+
+/**
+ * \ingroup FAPL
+ *
+ * \brief Modifies the file access property list to use the #H5FD_CORE driver
+ *
+ * \fapl_id
+ * \param[out] increment Size, in bytes, of memory increments
+ * \param[in] backing_store Size, in bytes, of memory increments
+ *             contents to disk when the file is closed
```

Comment:
This description is different from the current RM: "Boolean flag indicating whether to write the file contents to disk when the file is closed."